check-services.php

```diff
@@ -13,7 +13,7 @@
  * the source code distribution for details.
  */
 
-$init_modules = array();
+$init_modules = array('alerts');
```

Comment:
Need to remove 'alerts' now that this is no longer running alerting.

Reply:
Forgot about that. Done :)